gradoop-flink/src/main/java/org/gradoop/flink/algorithms/fsm/dimspan/functions/mining/VerifyPattern.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of Gradoop.
+ *
+ * Gradoop is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Gradoop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Gradoop. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.gradoop.flink.algorithms.fsm.dimspan.functions.mining;
+
+import org.apache.flink.api.common.functions.FilterFunction;
+import org.gradoop.flink.algorithms.fsm.dimspan.config.DIMSpanConfig;
+import org.gradoop.flink.algorithms.fsm.dimspan.gspan.GSpanLogic;
+import org.gradoop.flink.algorithms.fsm.dimspan.model.GraphUtils;
+import org.gradoop.flink.algorithms.fsm.dimspan.model.GraphUtilsBase;
+import org.gradoop.flink.algorithms.fsm.dimspan.model.Simple16Compressor;
+import org.gradoop.flink.model.impl.tuples.WithCount;
+
+/**
+ * DFS code => true, if minimal
```

Comment:
Is this where the magic happens? (That is, that the Isomorphism problem reveals itself? I'd provide trails from the Acutual algorithm to this place. I think this should one of the most interesting parts of the code)